system-tests/src/test/java/org/eclipse/dataspaceconnector/system/tests/local/BlobTransferIntegrationTest.java

```diff
@@ -16,86 +16,72 @@
 
 package org.eclipse.dataspaceconnector.system.tests.local;
 
-import com.azure.identity.DefaultAzureCredentialBuilder;
-import com.azure.security.keyvault.secrets.SecretClientBuilder;
+import com.azure.core.util.BinaryData;
+import com.azure.storage.blob.BlobContainerClient;
 import com.azure.storage.blob.BlobServiceClient;
-import com.azure.storage.blob.BlobServiceClientBuilder;
-import com.azure.storage.common.StorageSharedKeyCredential;
-import io.restassured.response.ResponseBodyExtractionOptions;
-import org.apache.commons.lang3.StringUtils;
-import org.eclipse.dataspaceconnector.system.tests.utils.TransferSimulationUtils;
-import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
 
-import java.util.Objects;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
 
-import static io.restassured.RestAssured.given;
 import static java.lang.String.format;
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.eclipse.dataspaceconnector.system.tests.local.BlobTransferLocalSimulation.ACCOUNT_NAME_PROPERTY;
-import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.API_KEY;
-import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.API_KEY_HEADER;
-import static org.eclipse.dataspaceconnector.system.tests.local.TransferLocalSimulation.CONSUMER_MANAGEMENT_URL;
-import static org.eclipse.dataspaceconnector.system.tests.utils.GatlingUtils.runGatling;
 import static org.eclipse.dataspaceconnector.system.tests.utils.TransferSimulationUtils.PROVIDER_ASSET_FILE;
-import static org.eclipse.dataspaceconnector.system.tests.utils.TransferSimulationUtils.TRANSFER_PROCESSES_PATH;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class BlobTransferIntegrationTest {
-    public static final String DST_KEY_VAULT_NAME = getEnv("CONSUMER_EU_KEY_VAULT");
-    public static final String BLOB_STORE_ENDPOINT_TEMPLATE = "https://%s.blob.core.windows.net";
-    public static final String KEY_VAULT_ENDPOINT_TEMPLATE = "https://%s.vault.azure.net";
+@EnabledIfEnvironmentVariable(named = "LOCAL_BLOB_TRANSFER_TEST", matches = "true")
```

Comment:
How about using a bit more generic variables like "TEST_ENVIRONMENT=LOCAL" and "TEST_ENVIRONMENT=CLOUD" in case we need to apply the same pattern to other tests?

Reply:
Looks good, updated. thanks